scripts/launch/connect-to-cluster/default.nix

```diff
@@ -82,6 +83,47 @@ let
       -H "Content-Type: application/json; charset=utf-8" \
       "https://${walletListen}/$request_path" "$@"
   '';
+  tlsConfigResultant = {
```

Comment:
We should keep the existing names of the CA. Keep what you have in the sample-wallet-config.nix though. I'm only referring to the defaults if you don't use a custom wallet config:
```
Issuer: O=Input Output HK, CN=Cardano SL Self-Signed Root CA
        Validity
            Not Before: Oct 18 02:47:21 2018 GMT
            Not After : Oct 18 02:48:21 2019 GMT
        Subject: O=Input Output HK, CN=Cardano SL Server Node

      Issuer: O=Input Output HK, CN=Cardano SL Self-Signed Root CA
        Validity
            Not Before: Oct 18 02:47:21 2018 GMT
            Not After : Oct 18 02:48:21 2019 GMT
        Subject: O=Input Output HK, CN=Daedalus Wallet

```

Reply:
@disassembler, done.

What is the rationale, though?

The certificate chain generation doesn't happen on IOHK's premises, or under IOHK control -- it's only guided by our scripts.  There's probably something to say about certificates relate to legal documents and what this entails.

I do understand the appeal (for the end-user) of having the vague authority of the certs stamped with our insignia, though.

I don't have a strong opinion anyway..

Reply:
> The certificate chain generation doesn't happen on IOHK's premises, or under IOHK control -- it's only guided by our scripts.

That's all true. IMHO the generation of these certs from scripts we maintain is sufficient for keeping the original identifiers. If users ever need to look at the audit the details of the TLS connection these identifiers will help them (and anyone they contact at IOHK) track down relevant context.

> There's probably something to say about how certificates relate to legal documents and what this entails.

If there are concerns about this you can consult with the legal team.